tests/other/ProtocolSettings.test.js

```diff
@@ -968,6 +968,76 @@ contract("ProtocolSettings", (accounts) => {
             expect(event["newRolloverFlexFeePercent"] == new_percent.toString()).to.be.true;
             expect((await sovryn.rolloverFlexFeePercent()).eq(new_percent)).to.be.true;
         });
+
+        it("Admin should not be able to set the admin address", async () => {
```

Comment:
missing a test for a random address not owner and not admin not being able to set admin

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/511/commits/f185515e59597af142af64b9dfded7b23ec023e8)